itest/rust/src/builtin_tests/geometry/quaternion_test.rs

```diff
@@ -28,4 +29,145 @@ fn quaternion_from_xyzw() {
     assert_eq!(quat.w, 0.8924);
 }
 
+#[itest]
+fn quaternion_from_axis_angle() {
+    // 1. Should generate quaternion from axis angle
+    let quat = Quaternion::from_axis_angle(Vector3::new(0.0, 0.0, 1.0).normalized(), 1.0);
+
+    // Taken from doing this in GDScript.
+    assert_eq!(quat.x, 0.0);
+    assert_eq!(quat.y, 0.0);
+    assert_eq_approx!(quat.z, 0.479426);
+    assert_eq_approx!(quat.w, 0.877583);
+    // 2. Should panic if axis is not normalized
```

Comment:
Small nitpick -- for each comment, make sure 
- there's an empty line before (at the start of a block)
- it starts with capital letter (unless it's a lowercase name/symbol)
- it ends with fullstop

Reply:
It would be nice if the formatter caught these. That way a robot can catch this before you've gotta read it 😂

Reply:
Rustfmt is not flexible enough (there are some nightly options that maintainers are too scared of stabilizing, but even those don't cover everything).

I would need to write a custom format script. Which I started doing once, but it would also need to be fast to compile, fast to run, etc. Will see...

Reply:
Yeah in terms of priorities it shouldn't even be mentioned as a thing that could hypothetically be prioritized. Would be nice though. 😂